packages/app/src/sandbox/eval/presets/vue-cli/index.js

```diff
@@ -148,9 +149,13 @@ export default function initialize() {
     { transpiler: noopTranspiler },
   ]);
   vuePreset.registerTranspiler(() => true, [{ transpiler: rawTranspiler }]);
-  vuePreset.registerTranspiler(m => m.path.endsWith('pug'), [
+  vuePreset.registerTranspiler(module => /\.pug$/.test(module.path), [
```

Comment:
I updated the `pug` section also, just so all sections in this file use the same syntax.